exporters/otlp/include/opentelemetry/exporters/otlp/otlp_file_client.h

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/sdk/common/exporter_utils.h"
+
+#include "opentelemetry/exporters/otlp/otlp_file_client_options.h"
+
+#include <chrono>
+#include <cstddef>
+#include <memory>
+
+// forward declare google::protobuf::Message
```

Comment:
This is not needed as it is already declared in the otlp_file_client_options.h included at line 9